src/lib/l1/tx.ts

```diff
@@ -35,26 +35,25 @@ export async function deposit(
           parseInt(amount),
           token_id
         );
+
         let l1_txhash = "";
-        r.when("snapshot", "Approve", () =>
-          progress("approve", "Wait confirm ...", "", 10)
```

Comment:
This is necessary, since if user does not confirm in  metamask, the hint should be waiting for confirm instead of SNAPSHOTAPPROVE
.